### ISSUE TITLE
migrate from middleman to Phoenix 

### ISSUE THREAD

**Issue by catherine-jones** [Elixir-Girls/elixirgirls.github.io#63]

**Comment by suranyami** [comment]
I'm happy to get started on this.

**Comment by suranyami** [comment]
@catherine-jones Just wondering: this is hosted on GitHub Pages, right? Which means they only do static site hosting?

I'm wondering, would we move the hosting off GitHub Pages to something like Heroku?

Or should I hunt down static site renderers that work with Phoenix instead?

Are there any plans to add more interactivity to the site, like having user login and/or forums, etc? If so, we should probably consider hosting on Heroku or similar.

If there's no intention of going down that path, then pursuing static site rendering is probably the right path ahead.

**Comment by catherine-jones** [comment]
I think we should move to Heroku. It makes more sense to me, what do you think?

**Comment by suranyami** [comment]
Yeah, sure. If we ever want to add anything requiring a database, then it makes sense.

Okay, that's good to know. I'll continue on, and document as much as I can along the way.

**Comment by suranyami** [comment]
You still in Sydney? Can catch up to make sure I'm going about things the right way, if you like?

**Comment by catherine-jones** [comment]
yep, awesome thanks. I think it makes sense for our app to use the same practices that we teach.

**Comment by catherine-jones** [comment]
yes I am but I have exams next week so super busy sorry 😞

**Comment by suranyami** [comment]
No prob. Plenty of foundational work to do for now... Good luck in your exams!
